roles/load_aware/load_aware_deploy_trimaran/tasks/main.yml

```diff
@@ -110,6 +110,11 @@
       oc describe pod trimaran-test -n trimaran > "{{ artifact_extra_logs_dir }}/trimaran-test.pod.desc"
       oc get all -n trimaran -oyaml > "{{ artifact_extra_logs_dir }}/all-trimaran.yaml"
 
+  - name: Dump trimaran scheduler log
+    shell: |
+      oc logs -n trimaran $(oc get pod -n trimaran -l "app=trimaran-scheduler" | awk 'NR > 1 {print $1}') > "{{ artifact_extra_logs_dir }}/trimaran_scheduler.log"
```

Comment:
Let's make this a bit cleaner and use `--no-headers` when listing the pods.  Then you'll not need to use 'NR > 1'.

Reply:
Will do!